modules/engage-theodul-plugin-tab-transcript/src/main/resources/static/styles/desktop.css

```diff
@@ -23,17 +23,10 @@
 
 #engage_transcript_tab #transcript_tab_search {
     width: 30%;
+    display: initial;
 }
 
 #engage_transcript_tab p {
     text-align: justify;
     margin-right: 1%;
 }
-
```

Comment:
Please add this to the bottom of the css so that the clear button is aligned with the search input field.
```
#engage_transcript_tab #clear_transcript_tab_search {
    position: relative;
    top: -2px;
}
```